packs/campaign-effects/resonant-reflection-reflection-of-life.json

```diff
@@ -30,6 +30,35 @@
                 "mode": "add",
                 "path": "system.attributes.hp.recoveryMultiplier",
                 "value": 1
+            },
+            {
+                "itemType": "action",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:slug:resonant-reflection"
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.ResonantReflection.ReflectionOfLife.Description"
+                    }
+                ]
+            },
+            {
+                "itemType": "action",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:slug:resonant-reflection"
+                ],
+                "property": "traits",
+                "value": "healing"
+            },
```

Comment:
I don't think we want to add the traits to the action, since it would get all of the traits at once.